recipes/hyperopt/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hyperopt" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/hyperopt/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: fa8d03f6663ecdd20ddec7e24aa1931cfd5ebf37a7881e90611dfe1d97b82e6f
+
+build:
+  number: 0
```

Comment:
I think this package could be ``noarch: python``

Reply:
Yes you are right. I added it. Thanks for the hint.